mm2src/coins/eth.rs

```diff
@@ -685,10 +685,11 @@ impl Deref for EthCoin {
 #[async_trait]
 impl SwapOps for EthCoin {
     fn send_taker_fee(&self, fee_addr: &[u8], amount: BigDecimal, _uuid: &[u8]) -> TransactionFut {
-        let address = try_fus!(addr_from_raw_pubkey(fee_addr));
+        let address = try_tx_fus!(addr_from_raw_pubkey(fee_addr));
 
         Box::new(
-            self.send_to_address(address, try_fus!(wei_from_big_decimal(&amount, self.decimals)))
+            self.send_to_address(address, try_tx_fus!(wei_from_big_decimal(&amount, self.decimals)))
+                .map_err(TransactionErr::PlainError)
```

Comment:
`EthCoin::send_to_address` should return `TransactionErr::TxRecoverableError`. Please look at the `sign_and_send_transaction_impl` function: 
https://github.com/KomodoPlatform/atomicDEX-API/blob/c23b1f7bb2941b51297709aee3ac3c46f5370c6f/mm2src/coins/eth.rs#L1382